sql/src/main/java/io/crate/action/sql/DDLStatementDispatcher.java

```diff
@@ -176,7 +172,7 @@ private Plan genCountStarPlan(TableInfo table) {
                 querySpec
                 );
         SelectAnalyzedStatement statement = new SelectAnalyzedStatement(queriedTable);
-        return planner.process(statement, new Planner.Context(clusterService));
+        return planner.process(statement, new Planner.Context(clusterService, UUID.randomUUID()));
```

Comment:
the DDLTask which uses the DDLStatementDispatcher already has a jobId which should be re-used imho